app/views/schools/dashboard/_school_cohort_details.html.erb

```diff
@@ -48,7 +48,7 @@
       <% else %>
         <% row.with_value(text: school_cohort_lead_provider_name(school_cohort)) %>
       <% end %>
-      <% row.with_action(text: :none) %>
+      <% row.with_action(text: 'Change', href: support_path(cohort_year: school_cohort.start_year, school_id: @school.id, subject: :"change-cohort-lead-provider")) %>
```

Comment:
Add some visually-hidden text here for screen-reader users (to avoid having 2 identical "Change" links).

```suggestion
      <% row.with_action(text: 'Change', visually_hidden_text: "lead provider", href: support_path(cohort_year: school_cohort.start_year, school_id: @school.id, subject: :"change-cohort-lead-provider")) %>
```